src/components/victory-line.jsx

```diff
@@ -253,6 +254,7 @@ const defaultProps = {
   scale: () => d3.scale.linear(),
   y: (x) => x,
   animate: false,
+  velocity: 0.02,
```

Comment:
I'm happy so long as this value matches the default in victory animation. Approved!